halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/security/authn/EditOAuth2Command.java

```diff
@@ -56,6 +57,13 @@
   )
   private String preEstablishedRedirectUri;
 
+  @DynamicParameter(
```

Comment:
How is this parsed? The above generated help text above doesn't make this too clear